linode/manager
# refactor: [M3-6804] - MUI v5 Migration - `Components > Chip`

## Description 📝
- Refactor for the `<Chip />` component
- Follow up to @abailly-akamai's https://github.com/linode/manager/pull/9310

## Major Changes 🔄
- Move  the `<Chip />` component out of the deprecated `core` folder 🗂️

## How to test 🧪
- Verify the Github Actions pass
- Double check the `<Chip />` within the App
- Double check the `<Chip />` within Storybook
